x-pack/plugins/apm/server/lib/connections/get_connection_stats/get_stats.ts

```diff
@@ -78,22 +157,14 @@ export const getStats = async ({
       },
       aggs: {
         connections: {
-          composite: {
-            size: 10000,
-            sources: asMutableArray([
+          multi_terms: {
```

Comment:
Why the change to multi_terms?

Reply:
Brain fart. I changed it because I removed the pagination.  According to the docs, composite is faster and more efficient in this case.